src/main/kotlin/com/exactpro/th2/lwdataprovider/grpc/GrpcUtils.kt

```
@@ -32,11 +35,12 @@ fun List<com.exactpro.th2.dataprovider.grpc.MessageSearchRequest.ResponseFormat>
     if (this.isNullOrEmpty())  {
         list.add(ResponseFormat.ALL)
     } else
-        for (i in this) {
-            when (i.name) {
-                ResponseFormat.ALL.name -> list.add(ResponseFormat.ALL)
-                ResponseFormat.BASE_64.name -> list.add(ResponseFormat.BASE_64)
-                ResponseFormat.PARSED.name -> list.add(ResponseFormat.PARSED)
+        for (responseFormat in this) {
+            when (responseFormat) {
+                ALL -> list.add(ResponseFormat.ALL)
+                BASE_64 -> list.add(ResponseFormat.BASE_64)
+                PARSED -> list.add(ResponseFormat.PARSED)
+                else -> {}
```

You should report an error in case of unknown format